packages/ripple-nuxt-tide/modules/webform/conditional-logic.js

```diff
@@ -147,19 +147,38 @@ function performTriggerCheck (rule) {
       if (typeof rule.triggerValue === 'string') {
```

Comment:
I guess this logic will filter out all number values. So new logic https://github.com/dpc-sdp/ripple/pull/894/files#diff-5667abf8c7db140ddc6fc57e232e1a5f2268574feb08a3db960b2b3d87baef40R150 won't be used?

Reply:
@tim-yao This logic filters out the value that is set in the BE, against which the entered Value is matched. The value that gets sends from the BE is always a string even if you set a number value. This is why I am converting the trigger value to a number to match with the entered number value.